action.yml

```diff
@@ -26,6 +26,12 @@ inputs:
   githubToken:
     description: 'The Github Token to be used to authenticate with Vault'
     required: false
+  roleName:
+    description: "The role name for the kubernetes authentification"
+    required: false
+  tokenPath:
```

Comment:
This should probably have a default `/var/run/secrets/kubernetes.io/serviceaccount/token`, which for 99% of the use cases the JWT will be found.

Reply:
gonna do that :)

Reply:
@jojo19893 Are you still planning on making this change? I can, to get this moving again

Reply:
Poke 😄